cmd/main.go

```diff
@@ -36,7 +45,41 @@ var (
 )
 
 var saramaLogger = log.New(io.Discard, "[Sarama] ", log.LstdFlags)
+
+func InitTracerProvider(ActivateTracing bool, JaegerExporterTracing bool, OtlpExporterTracing bool) *sdktrace.TracerProvider {
```

Comment:
is it needed to be exported in other packages? I see it's used in main only so lowercase `initTracerProvider` should be ok, or?